lib/pf/task/sendmail.pm

```diff
@@ -0,0 +1,82 @@
+package pf::task::sendmail;
+
+=head1 NAME
+
+pf::task::sendmail add documentation
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::task::sendmail
+
+=cut
+
+use strict;
+use warnings;
+use base 'pf::task';
+use POSIX;
+use Net::SMTP;
+use pf::util qw(untaint_chain);
+use pf::log;
+use pf::config;
+my $logger     = get_logger();
+
+sub doTask {
```

Comment:
This should call a util method instead of defining it here in case we need to call it directly